js/installversion.js

```diff
@@ -0,0 +1,52 @@
+var api = 'https://api.github.com/repos/CanonicalLtd/multipass/releases';
```

Comment:
It may not matter much for a small site like this, but it's a good practice not to expose variables into global scope if not necessary. Currently it seam that every function or variable is in global scope which may quickly lead to namespace clashes, unexpected overrides, etc.

Easiest way to avoid it without introducing build step would be to wrap the code into immediately called function.

Reply:
I agree with @bartaz as a general principle, but given the type of site this is I think this is acceptable.